translation-web/src/main/java/eu/europeana/api/translation/web/utils/PreProcessorUtils.java

```diff
@@ -0,0 +1,39 @@
+package eu.europeana.api.translation.web.utils;
+
+import eu.europeana.api.translation.definitions.model.TranslationObj;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class PreProcessorUtils {
+
+    /**
+     * Any value that has at least 2 unicode consecutive letters. The condition considered the
+     * fact that there can be words with only 2 letters that retain sufficient meaning and are therefore reasonable to be translated,
+     * especially when looking at languages other than English (see article - https://www.grammarly.com/blog/the-shortest-words-in-the-english-language/).
+     */
+    private static final String PATTERN = "\\p{IsAlphabetic}{2,}";
+    private static final Pattern isAlphabetic = Pattern.compile(PATTERN);
```

Comment:
it is better to not use the static access but to have the patterns in the beans. (one for translation and one for detection), the methods bellow blong actually to the business logic of the preprocessors